test/res.json.js

```diff
@@ -144,6 +144,28 @@ describe('res', function(){
         .expect(200, '{\n  "name": "tobi",\n  "age": 2\n}', done)
       })
     })
+
+    describe('"json escape" setting', function(){
+      it('should be undefined by default', function(){
+        var app = express();
+        assert(undefined === app.get('json escape'));
```

Comment:
I would really just say it's off by default and run through a response to validate it doesn't get escaped. That would help in case a change accidentally unconditionally escapes it.